src/docs/asciidoc/reference/configuration/structure-configuration.adoc

```diff
@@ -1,44 +1,59 @@
 :relative-path: ../../
 include::{docdir}/variables.adoc[]
 
-You can configure your applications with several variables. Here is the list of them:
-
-.Application variables
-[cols="5*^.^a", options="header"]
-|===
-| Name of the property
-| Definition
-| _package.json_
-| CLI parameter
-| Default value
-
-| Front code
-| 
-include::{docdir}/common/definitions/front-code.adoc[]
-| 
-include::{docdir}/common/code/front-property-package-json.adoc[]
-| --front <path>
-| `./front/`
-
-| Worker code
-| 
-include::{docdir}/common/definitions/worker-code.adoc[]
-| 
-include::{docdir}/common/code/worker-property-package-json.adoc[]
-| --worker <path>
-| `./worker/`
-
-| Worker entry point
-| 
-include::{docdir}/common/definitions/entry-point-worker.adoc[]
-| 
-include::{docdir}/common/code/entry-point-worker-package-json.adoc[]
-| _None_
-| `./worker/index.ts`
-
-
-
-|===
+It is possible to configure the structure of your application.
+In this part, we list the different possibilities.
+
+
+[#change-worker-entry-point]
+== Entry point for worker
+
+By default, the entry point of your worker code is the file `./worker/index.ts` to follow the NodeJS convention.
```

Comment:
not really a NodeJS convention to use worker folder. `index.ts` is more in convention but the real convention is index.js at root